### PR TITLE
More spelling fix

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
Found while crawling in the code, or docs

### CODE REVIEW

File: Configurations/README.design [hunk 1]
```diff
@@ -498,7 +498,7 @@ etc.
 
 Along with the build-file templates is the driving engine
 Configurations/common.tmpl, which looks through all the information in
-%unified_info and generates all the rulesets to build libraries,
+%unified_info and generates all the rules-set to build libraries,
```
Comment (richsalz) [inline Configurations/README.design]: nope, the original is correct, unless you want to replace with "rules" or "set of rules"

**Comment by kroeckx** [comment]
+1

**Comment by richsalz** [comment]
pushed with 79cc188 thanks!

**Comment by FdaSilvaYY** [comment]
It seems that you miss your push. 
I don't see this commit neither on git.openssl.org or Github .

**Comment by richsalz** [comment]
Sorry, wrong commit b6453a6

**Pull request closed without merging**